core/src/com/unciv/ui/worldscreen/mainmenu/WorldScreenMenuPopup.kt

```diff
@@ -137,6 +126,54 @@ class WorldScreenMenuPopup(val worldScreen: WorldScreen) : Popup(worldScreen) {
         multiplayerPopup.addCloseButton()
         multiplayerPopup.open()
     }
+
+    /** Shows the [Popup] with the map editor initialization options */
+    private fun openMapEditorPopup() {
+
+        close()
+        val mapEditorPopup = Popup(screen)
+
+        mapEditorPopup.addGoodSizedLabel("Map editor".tr()).row()
+
+        // Create a new map
+        mapEditorPopup.addButton("New map") {
+            UncivGame.Current.setScreen(NewMapScreen())
+            mapEditorPopup.close()
+        }
+
+        // Load the map
+        mapEditorPopup.addButton("Load map") {
+            val loadMapScreen = LoadMapScreen(null)
+            loadMapScreen.closeButton.isVisible = true
+            loadMapScreen.closeButton.onClick {
+                UncivGame.Current.setWorldScreen()
+                loadMapScreen.dispose() }
+            UncivGame.Current.setScreen(loadMapScreen)
+            mapEditorPopup.close()
+        }
+
+        // Edit the current map
+        val editThisMapButton = mapEditorPopup.addButton("Open current map") {
```

Comment:
I think this should actually be disabled always... I know it's cool to cheat in single player, but I think this makes it too easy... =/